.config/mise/config.toml

```diff
@@ -20,7 +20,7 @@ run = "cargo fmt --all"
 run = "cargo fmt --all -- --check"
 
 [tasks.start]
-run = "cargo run --bin ccc-server"
+run = "cargo run --bin ccc-server -- ${@}"
```

Comment:
Was this needed? I thought this was the default behavior if you did "Mise run start -- --help" or w/e

Reply:
In the first line, any args after run would be passed to `cargo run --bin ccc-server`, not to the binary itself.

In the second line, the `${@}` this captures any additional args and passes them, which tells cargo that these are meant for the program it's running and not for cargo itself.

Reply:
_oh_! That makes sense.